Source/objects.cpp

```diff
@@ -1427,7 +1427,7 @@ void AddFlameLvr(int i)
 }
 // 679768: using guessed type int trapid;
 
-void AddTrap(int i)
+void AddTrap(int i, int /*t*/)

```

Comment:
Why is the parameter name of `t` commented out? Because it is unused?

Reply:
Yes to avoid possible warnings, I don't know if it's that important to be honest.

Reply:
You can uncomment it, there are many functions elsewhere with unused args. Such as `random(int unused, BYTE idx)`

Reply:
Alright, uncommented.